inc/ticket.class.php

```diff
@@ -7136,14 +7206,12 @@ function getTicketActors() {
                       LEFT JOIN `glpi_users` usr ON gu.`users_id` = usr.`id`
                       WHERE gt.`tickets_id` = ".$this->getId()."
                      ) AS allactors
-                WHERE `type` != ".CommonItilActor::OBSERVER."
-                GROUP BY `users_id`
-                ORDER BY `type` DESC";
+                ";
 
       $res               = $DB->query($query);
       $ticket_users_keys = [];
       while ($current_tu = $DB->fetch_assoc($res)) {
-         $ticket_users_keys[$current_tu['users_id']] = $current_tu['type'];
+         $ticket_users_keys[$current_tu['users_id']][] = $current_tu['type'];
```

Comment:
i don't understand this change

Reply:
Needed to group users by assignment type: requester, observer, or technicians

Reply:
right, i saw it